frontend/src/localization/languages/ukrainian.json

```diff
@@ -127,6 +128,7 @@
     "Fire": "Звільнити",
     "Switch role": "Змітини роль",
     "company_owner": "власник компанії",
+    "company owner": "власник компанії",
     "hr": "рекрутер",
     "employee": "працівник",
```

Comment:
Why to insert "company owner" key twice?

Reply:
First one when we render role name on page. Second one when we rednder role name on modal window.

Reply:
Okay, I didn't notice underline